log4j-1.2-api/src/main/java/org/apache/log4j/spi/LocationInfo.java

```diff
@@ -16,24 +16,21 @@
  */
 package org.apache.log4j.spi;
 
-import java.io.Serializable;
 import java.util.Objects;
 
 /**
  * The internal representation of caller location information.
  *
  * @since 0.8.3
  */
-public class LocationInfo implements Serializable {
```

Comment:
I am not sure we ca remove these, since we need to maintain compatibility with Log4j 1.2.